pages/company/company.module.scss

```diff
@@ -0,0 +1,74 @@
+main {
+	.typo {
+		width: 421px;
+		display: flex;
+		justify-content: center;
+		align-items: center;
+		background-color: #fafafa;
+		text-align: center;
+		border-radius: 0 30px 30px 0;
+	}
+
+	.pic {
+		max-width: 600px;
+		margin-right: 800px;
+	}
+
+	.story {
+		display: flex;
+		flex-wrap: wrap;
+		padding: 4vw;
+		gap: 30px;
+		justify-content: space-around;
+		align-items: center;
+
+		.cards {
+			display: grid;
+			grid-template-areas:
```

Comment:
I dont think this `grid-template-areas` is needed anymore cuz you added this same CSS property through  `gridTempleAreas` prop in the `Box` component (Story.tsx line 41)